src/stylesheets/donate.css

```diff
@@ -6,183 +6,3 @@ main.donate {
   color: var(--color-da);
   background-color: white;
```

Comment:
Hey @coderbyheart ended up converting most of the css to tailwind.  Is this last `main.donate` style needed?